dapr/actor/runtime/actor.py

```diff
@@ -182,14 +177,14 @@ async def _save_state_internal(self):
         """
         await self._state_manager.save_state()
 
-    async def _fire_timer_internal(self, name: str) -> None:
+    async def _fire_timer_internal(self, timer_callback: str, timer_data: Any) -> None:
```

Comment:
timer_data should be `bytes` type ?

Reply:
I tested and the type for the example was str:

` time_callback is called - timer_state (type: <class 'str'>)`